apps/studio/components/layouts/AppLayout/NoticeBanner.tsx

```diff
@@ -1,124 +1,65 @@
-import { useSupabaseClient } from '@supabase/auth-helpers-react'
-import { useQuery } from '@tanstack/react-query'
 import { ExternalLink } from 'lucide-react'
 import { useRouter } from 'next/router'
 
 import { useParams } from 'common'
 import { useAppBannerContext } from 'components/interfaces/App/AppBannerWrapperContext'
 import { useProfile } from 'lib/profile'
 import { Button } from 'ui'
+import { useAuthConfigQuery } from 'data/auth/auth-config-query'
+import { isSmtpEnabled } from 'components/interfaces/Auth/SmtpForm/SmtpForm.utils'
 
-// [Joshen] For this notice specifically, just FYI
-// 1 month after 26th Jan we'll need to add some contextual information about this deprecation
-// in the database settings pooling config section, for projects created before September 27th 2023
+// This file, like AppBannerWrapperContext.tsx, is meant to be dynamic - update this as and when we need to use the NoticeBanner
+
+// [Joshen] As of 19th September 24, this notice is around custom SMTP
+// https://github.com/orgs/supabase/discussions/29370
+// Timelines TLDR:
+// - 20th September 2024: Email template customization no longer possible without setting up custom SMTP provider
+// - 24th September 2024: Projects without custom SMTP will have their custom email templates returned back to default ones
+// - 26th September 2024: If no custom SMTP, emails can only be sent to email addresses in your project's organization
+// We can probably look to disable this banner perhaps a month from 26th Sept 2024 - so maybe end of October 2024
 
 export const NoticeBanner = () => {
   const router = useRouter()
+  const { ref: projectRef } = useParams()
   const { isLoading: isLoadingProfile } = useProfile()
 
   const appBannerContext = useAppBannerContext()
-  const {
-    ipv6BannerAcknowledged,
-    pgbouncerBannerAcknowledged,
-    vercelBannerAcknowledged,
-    onUpdateAcknowledged,
-  } = appBannerContext
-
-  const supabase = useSupabaseClient()
-  const { ref: projectRef } = useParams()
+  const { authSmtpBannerAcknowledged, onUpdateAcknowledged } = appBannerContext
 
-  // [Alaister]: using inline queries here since this is temporary
-  const { data, isLoading: isLoadingIpv6Enabled } = useQuery(
-    ['projects', projectRef, 'pgbouncer-enabled'],
-    async ({ signal }) => {
-      let query = supabase.rpc('ipv6_active_status', { project_ref: projectRef }).returns<
-        {
-          pgbouncer_active: boolean
-          vercel_active: boolean
-        }[]
-      >()
+  const { data: authConfig } = useAuthConfigQuery({ projectRef })
+  const smtpEnabled = isSmtpEnabled(authConfig)
```

Comment:
We should also take into account if they are using the auth hook here. So either smtp is enabled OR they have the auth hook configured we should not show the banner

Reply:
this is specifically just the email hook right?